src/styles/bar.css

```diff
@@ -9,7 +9,7 @@
   font-size: 1rem;
 }
 
-@media (max-width: 64em) {
+@media (max-width: 63.9375rem) {
```

Comment:
can't we use `calc` here?

Reply:
what for? What kind of calculation you had in mind?

Reply:
like `calc(64rem - 1px)`? It seems less tricky thant 63.9375rem imho

Reply:
Well the `calc()` is more tricky as a matter of fact but more readable, I'll give you that.

If you prefer roud numbers, I prefer to go mobile first and and add the MQ `min-width: 64rem` instead.

Reply:
And indeed, why are you not doing that ? MQ is supported everywhere now, there is no need to have the desktop as default

Reply:
Because it was already like this and I don't mind not round values, especially when it's usually in a variable. I tried to get the stylus variable and put it in a CSS custom property to use it in the MQ but apparently the CSS spec (level 1) doesn't allow that (yet).

But let's do that.